dataland-frontend/src/components/pages/RequestData.vue

```diff
@@ -345,23 +368,58 @@ export default defineComponent({
     },
 
     /**
-     * Resets form to allow the user to make a new data request
+     * Go to companies page
      */
-    resetForm() {
-      this.acceptedCompanyIdentifiers = [];
-      this.rejectedCompanyIdentifiers = [];
-      this.selectedFrameworks = [];
-      this.identifiersInString = "";
-      this.identifiers = [];
-      this.postBulkDataRequestObjectProcessed = false;
-      this.submittingSucceded = false;
+    goToCompanies() {
+      void this.$router.push("/companies");
     },
 
     /**
-     * Go to companies page
+     * Opens a pop-up to warn the user that the session will expire soon and offers a button to refresh it.
+     * If the refresh button is clicked soon enough, the session is refreshed.
+     * Else the text changes and tells the user that the session was closed.
+     * @param responseData returned data from the request to pass to dialog
      */
-    goToCompanies() {
-      void this.$router.push("/companies");
+    openRequestModal(responseData: BulkDataRequestResponse): void {
+      const rejectedIdentifiersCount = responseData.rejectedCompanyIdentifiers.length;
+
+      this.$dialog.open(this.bulkDataResponseDialog, {
+        props: {
+          modal: true,
+          closable: true,
+          closeOnEscape: true,
+          showHeader: true,
+          header: `${rejectedIdentifiersCount} identifier${rejectedIdentifiersCount > 1 ? "s" : ""} can't be found.`,
+          style: {
+            width: "50vw",
+          },
+          breakpoints: {
+            "1199px": "75vw",
+            "575px": "90vw",
+          },
+        },
+        data: {
+          onItemChangeHandler: (eventName: "removed" | "undo", identifier: string) =>
+            this.handleIdentifierChange(eventName, identifier),
+          responseData,
+        },
+      });
+    },
+
+    /**
+     * @param eventName whether "removed" or "undo"
```

Comment:
Description missing

Reply:
Done.

Reply:
For the handleIdentifierChange i only the params part, but no actual description of the method

Reply:
Removed to different branch ready for Phase 2

Reply:
ok